packages/UI/src/components/progressBar/styles.css

```diff
@@ -0,0 +1,21 @@
+@import '@radix-ui/colors/black-alpha.css';
```

Comment:
why importing radix colors? we only use the colors provided by our custom tailwind config

Reply:
yeah just for testing, I will fix it.